index.html

```diff
@@ -2,132 +2,134 @@
-            if (params.has('commit')) {
-                targetCommit = params.get('commit');
-                console.log('Using commit: ' + targetCommit);
-		document.title = document.title + ' commit: ' + targetCommit;
-            } else if (params.has('user') || params.has('branch') || params.has('repo') || latest) {
-                // Query github API
-                let api_url = `https://api.github.com/repos/${user}/${repo}/commits/${branch}`;
-                try {
-                    let res = await query_internal(api_url);
-                    targetCommit = JSON.parse(res.responseText)['sha'];
-		    if (params.has('branch'))
-			document.title = document.title + ' - ' + branch;
-                    console.log(`Using latest commit from ${user}/${repo} on branch ${branch}: ${targetCommit}`);
-                } catch (e) {
-                    console.log(`Failed to query Github API. Send: ${api_url} Response: ${res}`);
-                }
-            }
+        upstreamUrl = `https://raw.githubusercontent.com/${user}/${repo}/${targetCommit}/BondageClub/`;
 
-            upstreamUrl = `https://raw.githubusercontent.com/${user}/${repo}/${targetCommit}/BondageClub/`;
-
-            let index = await query('index.html');
-            let sourceScripts = index.responseText.split('\n')
-                .map(x => /<script src="([\w\d/._]+)"><\/script>/.exec(x))
-                .filter(x => x !== null)
-                .map(x => x[1])
-                .filter(x => !x.startsWith('Screens/Room/')); // Easy load time optimization
-            
-            sourceScripts.splice(sourceScripts.indexOf('Screens/Character/ItemColor/ItemColor.js'), 0, 'src/common-shim.js');
-            sourceScripts.push('src/index-shim.js', 'src/shim.js');
-
-            let styleSrc = await query('CSS/Styles.css');
-            let style = document.createElement('style');
-            style.innerHTML = styleSrc.responseText;
-            document.head.appendChild(style);
-
-            let sources = await Promise.all(sourceScripts.map(query));
-            sources.forEach(req => {
-                let script = document.createElement('script');
-                script.innerHTML = req.responseText;
-                document.body.appendChild(script);
-            });
-
-            init();
-        }
+        let index = await query('index.html');
+        let sourceScripts = index.responseText.split('\n')
+            .map(x => /<script src="([\w\d/._]+)"><\/script>/.exec(x))
+            .filter(x => x !== null)
+            .map(x => x[1])
+            .filter(x => !x.startsWith('Screens/Room/')); // Easy load time optimization
 
-        Click = () => {};
-        TouchStart = () => {};
-        MouseMove = () => {};
-        LoseFocus = () => {};
+        sourceScripts.splice(sourceScripts.indexOf('Screens/Character/ItemColor/ItemColor.js'), 0, 'src/common-shim.js');
+        sourceScripts.push('src/index-shim.js', 'src/shim.js');
 
-        load().then(
```

Comment:
Won't removing this cause problems? Am not good web dev sorry :S

Reply:
That was from me moving a few lines to another file